deploy.sh

```diff
@@ -1,49 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Not sure if we should remove this file because the Herokuhoosting should be a temporary thing. Perhaps we could keep it and leave a comment as this is not being currently used 🤔

Reply:
Yeah sure will edit @isabelcosta :v: